src/main.js

```diff
@@ -93,12 +90,17 @@ if (
 
 app.component("inline-svg", InlineSvg);
 app.use(i18n);
-app.use(PrimeVue, { ripple: true });
+app.use(VueTippy, {
+  // https://vue-tippy.netlify.app/installation
```

Comment:
why is this added?

Reply:
for reference on how to import this plugin